packages/keystatic/src/form/fields/markdoc/editor/editor-footer.tsx

```diff
@@ -0,0 +1,218 @@
+          </pre>
+
+          <h3>Styling text</h3>
+          <p>
+            You can indicate emphasis with bold, italic, or strikethrough text.
+          </p>
+          <table>
+            <thead>
+              <tr>
+                <th scope="col">Style</th>
+                <th scope="col">Syntax</th>
+                <th scope="col">Shortcut</th>
+                <th scope="col">Example</th>
+                <th scope="col">Output</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Bold</td>
+                <td>
+                  <code>** **</code>
+                </td>
+                <td>
+                  <Kbd meta trim={false}>
+                    B
+                  </Kbd>
+                </td>
+                <td>
+                  <code>**This is bold text**</code>
+                </td>
+                <td>
+                  <strong>This is bold text</strong>
+                </td>
+              </tr>
+              <tr>
+                <td>Italic</td>
+                <td>
+                  <code>_ _</code>
+                </td>
+                <td>
+                  <Kbd meta trim={false}>
+                    I
+                  </Kbd>
+                </td>
+                <td>
+                  <code>_This text is italicized_</code>
+                </td>
+                <td>
+                  <em>This text is italicized</em>
+                </td>
+              </tr>
+              <tr>
+                <td>Strikethrough</td>
+                <td>
+                  <code>~~ ~~</code>
+                </td>
+                <td>None</td>
+                <td>
+                  <code>~~This was mistaken text~~</code>
+                </td>
+                <td>
+                  <del>This was mistaken text</del>
+                </td>
+              </tr>
+              <tr>
+                <td>Nested&nbsp;emphasis</td>
+                <td>
+                  <code>**&nbsp;**</code> and <code>_&nbsp;_</code>
+                </td>
+                <td>None</td>
+                <td>
+                  <code>**This text is _extremely_ important**</code>
+                </td>
+                <td>
+                  <strong>
+                    This text is <em>extremely</em> important
+                  </strong>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+
+          <h3>Lists</h3>
+          <p>
+            You can make an unordered (bulleted) list by preceding one or more
+            lines of text with <kbd>-</kbd>, or <kbd>*</kbd>.
+          </p>
+          <pre>
+            <code>{`- Red\n- Green\n- Blue`}</code>
+          </pre>
+
+          <p>
+            Create an ordered (numbered) list by preceding each line with a
+            number.
+          </p>
+          <pre>
+            <code>{`1. Red\n1. Green\n1. Blue`}</code>
+          </pre>
```

Comment:
Not sure I'd describe it like this, seems a bit weird since you'd just add e.g. the `1.` on the first time and then press enter for more, you wouldn't precede every line with a number yourself

Reply:
True, needs to be reconsidered.